### DISCUSSION TITLE
Upgrade from 3.4.0 to latest - docs are outdated or what's the latest stable?

### DISCUSSION THREAD

**Discussion by xairoo** [simple-login/app#1742] [Self-hosting question] [answered]
Looks like the doc is outdated, everything points to 3.4.0: https://github.com/simple-login/app/blob/master/docs/upgrade.md

What's the best way to upgrade to the latest stable v4 build?
Replacing `:3.4.0` in the upgrade doc just with `:latest`?

Or is 3.4.0 the last stable? I am bit confused =)

Thanks!

**Comment by t2noob** [comment, marked as answer]
you can use the latest self hosted version and upgrade https://github.com/simple-login/app/blob/new-self-host-version/docs/upgrade.md 
Or use the same upgrade directions but replace with the app-ci from docker hub

**Reply by xairoo** [reply]
This worked =)
I was confused because the master branch still describes 3.4.0.
Thank you!

**Reply by t2noob** [reply]
NP glad to help

**Comment by mfld-pub** [comment]
I am unsure if I am using the right branch, too.

> you can use the latest self hosted version and upgrade https://github.com/simple-login/app/blob/new-self-host-version/docs/upgrade.md

If I do that, I just end up with the same 4.6.5-beta I had a few months ago

```
CONTAINER ID   IMAGE                        
f63843cea2bf   simplelogin/app:4.6.5-beta
```

`docker pull simplelogin/app:4.6.5-beta`

Should this be `docker pull simplelogin/app:new-self-host-version`  instead ?

**Reply by dasrohr** [reply]
you are still pulling the image simplelogin/app which has the latest version tagged with 4.6.5-beta.

there is an image at simplelogin/app-ci which has that tag 4.35.1.

if I understand correctly you can just pull this with the update instructions from the document above.

haven't done that myself though. so take it with a grain of salt.
but I am going to try this when I have some spare time as I am also running 3.4.0 atm. I was also falsely reading the upgrade docs from the master branch like the OP did and being confused about the stale version.

**Reply by dasrohr** [reply]
I just came across this post:
https://github.com/simple-login/app/discussions/1831#discussioncomment-6690081

so ... not really sure if switching the app-ci image on a working production environment is a good idea. make sure to have a backup if trying this or just stick with what the doc says 4.6.5-beta

**Reply by dasrohr** [reply]
but also, there is this post 

https://github.com/simple-login/app/discussions/1831#discussioncomment-6690081

so ... this still leaves me a bit confused but using app-ci might be just fine after all.
